### PR TITLE
Dependency updates/cleanup

### PR DESCRIPTION
**Pull request by bertfrees**

### CODE REVIEW

**Comment by bertfrees** [comment]
@rdeltour Forgot to mention this depends on a change in epubcheck: https://github.com/w3c/epubcheck/compare/master...daisy:super/dependencies.

File: bom/pom.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
       <dependency>
         <groupId>org.idpf</groupId>
         <artifactId>epubcheck</artifactId>
-        <version>4.0.1</version>
+        <version>4.0.3-SNAPSHOT</version>
```
Comment (rdeltour) [inline bom/pom.xml]: epubcheck v4.1.0 will be released very soon (no later than 3d week of November), maybe we can wait for it and include it in this PR?

**Comment by bertfrees** [comment]
Merged everything except the epubcheck part in 85ff94a.

File: bom/pom.xml (hunk 1 above)
Comment (bertfrees) [reply bom/pom.xml]: OK. I'll make a PR from https://github.com/w3c/epubcheck/compare/master...daisy:super/dependencies then.

**Comment by bertfrees** [comment]
https://github.com/w3c/epubcheck/pull/918

**Comment by bertfrees** [comment]
epubcheck part merged in bd16b9b

**Pull request closed without merging**